src/tool_cb_hdr.c

```diff
@@ -401,8 +404,8 @@ void write_linked_location(CURL *curl, const char *location, size_t loclen,
      !strcmp("https", scheme) ||
      !strcmp("ftp", scheme) ||
      !strcmp("ftps", scheme)) {
-    fprintf(stream, LINK "%s" LINKST "%.*s" LINKOFF,
-            finalurl, (int)loclen, location);
+    fprintf(stream, " " LINK "%s" LINKST "%.*s" LINKOFF,
+            finalurl, (int)prevllen, loc);
```

Comment:
why should this use `prevllen` and not `locclen`? The first one includes whitespace and the latter does not.